Userland/Libraries/LibC/pthread.cpp

```diff
@@ -553,6 +553,12 @@ int pthread_cancel(pthread_t thread)
     return pthread_kill(thread, SIGCANCEL);
 }
 
+// https://pubs.opengroup.org/onlinepubs/009695399/functions/pthread_testcancel.html
+void pthread_testcancel(void)
+{
+    // TODO(cwasser): Stub to link the Qt Concurrent module
```

Comment:
I believe pthread_cancel is implemented more or less completely, and this function should call `__pthread_maybe_cancel`.

Reply:
Thanks!